src/ODEs/Exports.jl

```diff
@@ -11,6 +11,8 @@ end
 @publish_gridapodes ODETools ThetaMethod
 @publish_gridapodes ODETools RungeKutta
 @publish_gridapodes ODETools IMEXRungeKutta
+@publish_gridapodes ODETools EXRungeKutta
+@publish_gridapodes ODETools DIRungeKutta
```

Comment:
better first EX -> accept
then we start with DI

Reply:
removed in clean version